nomad/resource_job_test.go

```diff
@@ -305,22 +367,40 @@ func testResourceJob_updateCheck(s *terraform.State) error {
 	return nil
 }
 
-var testResourceJob_parameterizedJob = `
+func TestResourceJob_vault(t *testing.T) {
+	r.Test(t, r.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []r.TestStep{
+			{
+				Config: testResourceJob_vaultConfig,
+				Check:  testResourceJob_initialCheck,
```

Comment:
Is there any way we could check the output for Vault-specific things, to ensure the token was actually used?